external/metall_graph_construction_bench/README.md

```diff
@@ -0,0 +1,54 @@
+
+# Graph Construction Benchmark
+
+[Metall](https://github.com/LLNL/metall) contains a shared-memory dynamic graph construction benchmark.
+In this page, we describe how to run the benchmark with UMap (Metall can use UMap instead of system mmap() to allocate memory into persistent memory).
+
+In this benchmark, edges are added to a graph data structure to create a graph. More details about the benchmark are written in [this paper (see Section III)](https://www.osti.gov/servlets/purl/1576900).
+
```

Comment:
The cited URL doesn't give access to the paper or state where the paper was published (which conference) so that people can get it.

Reply:
It shows those information
https://www.osti.gov/servlets/purl/1576900

Also, I prefer to use the URL because it is free and open to everyone (unlike IEEE or ACM's pages).